platforms/gke/base/core/deploy.sh

```diff
@@ -17,6 +17,10 @@ set -e
 
```

Comment:
Nit (style): consider using the long format `set -o errexit` instead of the shorter `set -e` for more clarity in scripts where we don't need the shorter version to type faster :)

Reply:
(if we go this route, it's probably worth doing this in the `teardown.sh` script as well.)